azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/sqlserver/KeyVaultProperties.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.sqlserver;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.util.Assert;
+import org.springframework.validation.annotation.Validated;
+
+import javax.annotation.PostConstruct;
+import javax.validation.constraints.NotEmpty;
+
+@Getter
+@Setter
+
+@ConfigurationProperties("azure.sqlserver.keyvault")
```

Comment:
Does always encrypted only apply for Sql Server? If it could also be used for other database, it's better to be named `spring.datasource.encrypt.keyvault`

Reply:
It's SQL server specific, no other datasources support it